database/migrations/2023_04_21_232353_create_bucketlist_tables.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('bucketlists', function (Blueprint $table) {
+            $table->string('id')->primary();
+            $table->json('title')->nullable();
+            $table->timestamps();
+        });
+
+        Schema::create('bucketlist_item', function (Blueprint $table) {
+            $table->string('bucketlist_id');
+            $table->string('item_id');
+            $table
+                ->foreign('bucketlist_id')
+                ->references('id')
+                ->on('bucketlists');
+            $table
+                ->foreign('item_id')
+                ->references('id')
+                ->on('items');
+            $table->integer('ord');
```

Comment:
this might cause troubles on deleting items (`Integrity constraint violation`)
I suggest to include also delete cascade in pivot table when deleting the related record
(same issue was with `item_section` -> https://github.com/SlovakNationalGallery/atlas.sng.sk/pull/146 )

```suggestion
            $table
                ->foreign('bucketlist_id')
                ->references('id')
                ->on('bucketlists')
                ->onDelete('cascade');
            $table
                ->foreign('item_id')
                ->references('id')
                ->on('items')
                ->onDelete('cascade');
            $table->integer('ord');
```